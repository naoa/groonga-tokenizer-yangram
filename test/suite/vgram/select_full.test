register tokenizers/yangram

table_create vgram_words TABLE_HASH_KEY ShortText
load --table vgram_words
[
{"_key": "今日"},
{"_key": "日は"},
{"_key": "は雨"},
{"_key": "雨だ"}
]

table_create Memos TABLE_NO_KEY
column_create Memos content COLUMN_SCALAR ShortText

table_create Terms TABLE_PAT_KEY ShortText \
  --default_tokenizer TokenYaVgram \
  --normalizer NormalizerAuto
column_create Terms memos_content COLUMN_INDEX|WITH_POSITION Memos content

load --table Memos
[
{"content": "今日は雨だ"}
]

select Memos --match_columns content --query "今日は雨" --match_escalation_threshold -1
