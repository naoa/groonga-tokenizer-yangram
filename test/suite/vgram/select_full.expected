register tokenizers/yangram
[[0,0.0,0.0],true]
table_create vgram_words TABLE_HASH_KEY ShortText
[[0,0.0,0.0],true]
load --table vgram_words
[
{"_key": "今日"},
{"_key": "日は"},
{"_key": "は雨"},
{"_key": "雨だ"}
]
[[0,0.0,0.0],4]
table_create Memos TABLE_NO_KEY
[[0,0.0,0.0],true]
column_create Memos content COLUMN_SCALAR ShortText
[[0,0.0,0.0],true]
table_create Terms TABLE_PAT_KEY ShortText   --default_tokenizer TokenYaVgram   --normalizer NormalizerAuto
[[0,0.0,0.0],true]
column_create Terms memos_content COLUMN_INDEX|WITH_POSITION Memos content
[[0,0.0,0.0],true]
load --table Memos
[
{"content": "今日は雨だ"}
]
[[0,0.0,0.0],1]
select Memos --match_columns content --query "今日は雨" --match_escalation_threshold -1
[
  [
    0,
    0.0,
    0.0
  ],
  [
    [
      [
        1
      ],
      [
        [
          "_id",
          "UInt32"
        ],
        [
          "content",
          "ShortText"
        ]
      ],
      [
        1,
        "今日は雨だ"
      ]
    ]
  ]
]
